example/App.js

```diff
@@ -4,46 +4,97 @@
  * @flow
  */
 
-import React, { Component } from 'react';
+import React, { Component } from 'react'
+import { Platform, StyleSheet, Text, View, Button, Picker } from 'react-native'
```

Comment:
@ex7r3me looks like none of `Platform/Button/Picker` is being used in this file, so let's remove them :)

Reply:
Thanks, they have been removed